### PR TITLE
Unintended effects of SAFE Pro Rata Rights

### PR DESCRIPTION
**Pull request by hkhanna**
_Tl;dr: YC's post-money SAFE obviates the need to allow for pro rata rights in issuances in SAFEs and Convertible Notes (as opposed to preferred stock). Indeed, by offering pro rata rights in convertible security issuances, Bloomberg Beta has the opportunity to increase its percentage ownership above the negotiated percentage._

This PR is an analysis and some gentle suggestions regarding the Bloomberg Beta SAFE side letter. I assume for purposes of this PR, that SAFE investments by Bloomberg Beta take the form of the most recent YC "Post-Money" SAFE available on [its website](https://www.ycombinator.com/documents).

I suspect some of what is in this repo was written before YC updated its SAFE to be a "post-money" SAFE rather than a "pre-money" SAFE that was the standard until recently.

From the Bloomberg Beta README: 

> The challenge is that SAFE holders get pro rata rights only in the round *after* they become preferred stockholders of the company. Y Combinator’s new side letter grants these rights in the equity financing when they convert, which is a step in the right direction, but **SAFE holders still get diluted when companies issue SAFEs or convertible notes before the next equity round**.

I don't believe the bolded is true any longer. Indeed, it was the same problem you are solving for that was the impetus for YC to release the "postmoney" SAFE, which is "post" all converting instruments, but "pre" the equity financing. Quoting from the [YC user guide](https://www.ycombinator.com/assets/ycdc/Primer%20for%20post-money%20safe%20v1.1-2af8129e12effd9638eeab383b7309142c8f415e5cdb0bc210d573f779177a1c.pdf):

> The Post-Money Valuation Cap is “post” all of the safe money. It is NOT also “post” the Equity
Financing (e.g. Series A) money.

## Example

This example lays out a possible scenario for the way the existing Bloomberg Beta SAFE Side Letter is currently worded. Let's assume for this scenario that your pro rata right is 10% and we write that number in on the Bloomberg SAFE Side Letter.

1. Bloomberg Beta invests $100,000 into the Company on a $1,000,000 post-money valuation using a SAFE and their side letter. You own exactly 10% of the company on a fully-diluted basis.  

2. [Ice Cream Sandwich Capital](https://twitter.com/hkhanna/status/1586884097137414151) later also invests $100,000 into the company on a $1,000,000 post-money valuation also using a SAFE. 
    - Ice Cream Sandwich Capital owns 10% of the company (fully-diluted).
    - You _still own_ 10% of the company (fully-diluted). This is because when your SAFE converts at a later equity financing, it converts at a price per share which is reduced to reflect Ice Cream Sandwich's investment and any other "Converting Securities" (defined in the YC SAFE document). 

3. You exercise your pro rata right to participate in the Ice Cream Sandwich SAFE sale. This means you get to invest $10,000 on the $1,000,0000 post-money SAFE. **You now own _11%_ of the company on a fully-diluted basis, which I don't think was the intention.** 

This PR is intended to remove that right to invest alongside non-stock issuances of equity. The conversion mechanics in the SAFE itself should prevent you from being diluted by other converting securities, so a pro rata right there would only allow you to increase your percentage stake.

I hope this makes sense. I also very much could have misapprehended the purpose of the documents or even just plain read something wrong!

### CODE REVIEW

File: 02-SAFE/Side Letter.md [hunk 1]
```diff
@@ -15,33 +15,31 @@ Ladies and Gentlemen:
...
 
     (c) Promptly after the commencement thereof, notice of all suits, claims, proceedings or investigations that could materially and adversely affect the Company or any of its affiliates or with respect to the SAFE.
 
-5. (a) The Company agrees that, until the earlier of (i) conversion of the SAFE pursuant to the closing of the next Equity Financing contemplated by the SAFE or (ii) immediately prior to the consummation of a Change of Control of the Company, if the Company shall offer any investment terms to another investor that are more favorable than those set forth in the SAFE, the same terms shall be automatically deemed to apply retroactively to any and all investments in the Company by Investor that do not have such investment terms that are more favorable, whether pursuant to the SAFE or otherwise.  The Company agrees to notify Investor promptly upon proposing to or receiving from another investor terms that are more favorable to the investor than those set forth in the SAFE, as supplemented by this Agreement.
```
Comment (hkhanna) [inline 02-SAFE/Side Letter.md:32]: All of these changes are just renumbering the sections. This isn't a problem with Magistrate templates since one just uses the `{% section %}` tag and it auto-calculates the numbering.

**Comment by loriberenberg** [comment]
Harry, thanks for your analysis of our SAFE side letter template, and apologies for the delay in our response as we circled up with our legal team to give your note a thorough review. We include a fixed pro rata percentage in the side letter to be as simple and transparent as possible so founders can avoid costly calculations and chatter between legal teams. We want founders to know exactly what they’re negotiating so we opt to include a straightforward percentage; we do not use it as a workaround to increase our ownership. We've yet to ever encounter a situation like the one you outlined. And in practice, if the situation you outlined were to come up, we aim to act with best intentions and do what's right as opposed to insisting on contractual terms. For example, we frequently stand down on pro rata when founders explicitly ask us to, in favor of founder service.
